store/store_test.go

```diff
@@ -653,38 +653,75 @@ const MockSearchJSON = `{
 }
 `
 
-func (t *remoteRepoTestSuite) TestUbuntuStoreFind(c *C) {
+func (t *remoteRepoTestSuite) TestUbuntuStoreFindQueries(c *C) {
+	n := 0
 	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Here is what I had in mind: http://paste.ubuntu.com/19162143/ - not sure if it really adds clarity though.